<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmEdit</class>
 <widget class="QDialog" name="FilmEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="genreEdit">
     <property name="text">
      <string>Genre1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="actorsEdit">
     <property name="text">
      <string>1;2;3</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="actorsLabel">
     <property name="text">
      <string>ID актеров через ;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="nameEdit">
     <property name="text">
      <string>Film name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Сбросить</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="directorEdit">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="directorLabel">
     <property name="text">
      <string>ID режиссера</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="documentTitle">
      <string/>
     </property>
     <property name="markdown">
      <string>Film description

</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Film description&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="genreLabel">
     <property name="text">
      <string>Жанр</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="ratingEdit">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ratingLabel">
     <property name="text">
      <string>Рейтинг</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Длительность (мин)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="durationEdit">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
